### PR TITLE
update Closure Compiler to pipe to ./dist

### PR DESCRIPTION
**Pull request by samthor**
For unknown reasons, this works for me now.

### CODE REVIEW

**Comment by samthor** [comment]
@sgomes

File: gulpfile.js [hunk 1]
```diff
@@ -233,7 +232,8 @@ gulp.task('closure', function() {
         warning_level: 'VERBOSE'
         // jscs:enable closureCamelCase
       }
-    }));
+    }))
+    .pipe(gulp.dest("./dist"));
```
Comment (surma) [inline gulpfile.js]: Drone/JSHint is complaining because double quotes instead of single quotes

Comment (samthor) [inline gulpfile.js]: d'oh. Fixed, amended, pushed.

**Comment by surma** [comment]
LGTM! Thanks @samthor

**Comment by sgomes** [comment]
Sweet, thanks @samthor! LGTM, merging.

**Pull request merged**